catalog-info.yaml

```
@@ -322,11 +321,11 @@ spec:
           build.pull_request.id == null || (build.creator.name == 'elasticmachine' && build.pull_request.id != null)
       repository: elastic/beats
       cancel_intermediate_builds: true
-      cancel_intermediate_builds_branch_filter: "!main !7.17 !8.*"
+      cancel_intermediate_builds_branch_filter: "!main !7.* !8.*"
       skip_intermediate_builds: true
-      skip_intermediate_builds_branch_filter: "!main !7.17 !8.*"
-      # env:
-      #   ELASTIC_PR_COMMENTS_ENABLED: "true" TODO: uncomment when pipeline is ready
+      skip_intermediate_builds_branch_filter: "!main !7.* !8.*"
```

no need for supporting any 7.x but 7.17 
```suggestion
      skip_intermediate_builds_branch_filter: "!main !7.17 !8.*"
```

---

That won't be the case. `7.17` is the last branch for `7.x` - hence no `7.18` branch will be created.  https://www.elastic.co/support_policy explains what's the support.